packages/fluent-theme/src/fluent/styles/SearchBox.styles.ts

```diff
@@ -1,14 +1,44 @@
 import { ISearchBoxStyleProps, ISearchBoxStyles } from 'office-ui-fabric-react/lib/SearchBox';
 
 export const SearchBoxStyles = (props: ISearchBoxStyleProps): Partial<ISearchBoxStyles> => {
-  const { theme } = props;
+  const { theme, underlined } = props;
   const { effects, palette } = theme;
 
   return {
     root: [
       {
         borderRadius: effects.roundedCorner2,
-        borderColor: palette.neutralSecondaryAlt
+        borderColor: palette.neutralTertiary
+      },
+      underlined && [
+        'is-underlined',
```

Comment:
No need for this class to be added in here as it's already in the default styles. Only provide the things you want to override.